### PR TITLE
Configure separate queues for DSA robots.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made? 🤔
So that specific robots can be run within DSA.

## How was this change tested? 🤨
Unit, integration test

⚡ ⚠ If this change affects consumers, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** that exercise this service and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: app/models/workflow_step.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class WorkflowStep < ApplicationRecord
   validates :workflow, presence: true
   validates :process, presence: true
   validates :version, numericality: { only_integer: true }
-  validates :status, inclusion: { in: %w[waiting started completed queued error skipped] }
+  validates :status, inclusion: { in: %w[waiting started completed queued error skipped retrying] }
```
Comment (justinlittman) [inline app/models/workflow_step.rb:9]: The status of a workflow step will be changed to "retrying" will the job is retrying.

File: app/services/queue_service.rb [hunk 2]
```diff
@@ -26,6 +26,8 @@ def enqueue
 
   private
 
+  DSA_ROBOTS = ['Robots::DorRepo::Accession::Publish'].freeze
```
Comment (justinlittman) [inline app/services/queue_service.rb:29]: More will be added here.

File: app/models/workflow_step.rb (hunk 1 above)
Comment (lwrubel) [reply app/models/workflow_step.rb:9]: How is "retrying" set as a status? Is that to be implemented?

Comment (lwrubel) [reply app/models/workflow_step.rb:9]: Oh, I see, that's in lyber-core.

**Review by lwrubel — APPROVED** [review]

**Pull request merged**